frappe/commands/utils.py

```diff
@@ -768,25 +768,45 @@ def set_config(context, key, value, global_=False, parse=False, as_dict=False):
 
 
 @click.command('version')
-def get_version():
+@click.option('--output', help='Output format. One of: plain, table, json, legacy.', default='legacy')
```

Comment:
Add aliases `--format`, `-f` for the sake of consistency (wrt `list-apps`)? 
I think we can have both. Maybe add `--output` as an alias in `list-apps` too.